mantidproject/mantid
# Add group asymmetry estimate to new GUI

**Description of work.**

This PR adds the estimation of group asymmetry values to the new GUI. It also adds the ability to select the ranges used to estimate the group asymmetries to the grouping tab

**Report to:** anthony 

**To test:**

Open up the new GUI and load some data. Check that a group asymmetry estimate is calculated. Change the ranges on the grouping tab and check that the ranges used to estimate the group asymmetry are updated appropriately. Check that it also works for multiperiod and co-added data.



Fixes #24646  



*This does not require release notes* because the new GUI is not currently available to users


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
